helios-services/src/main/java/com/spotify/helios/servicescommon/ServiceParser.java

```diff
@@ -303,6 +309,11 @@ public String getZooKeeperAclAgentUser() {
     return kafkaBrokers.isEmpty() ? null : kafkaBrokers;
   }
 
+  public List<String> getPubsubPrefixes() {
+    final List<String> pubsubPrefixes = options.getList(pubsubTopicPrefixArg.getDest());
+    return pubsubPrefixes.isEmpty() ? null : pubsubPrefixes;
```

Comment:
i see from above that this is following the existing pattern but it feels weird to me that this returns null rather than an empty list, especially when the provider.senders() method has logic to return an empty list when nothing is enabled.